pkg/filters/redirectorv2/redirectorv2.go

```diff
@@ -76,7 +77,6 @@ type (
 	Spec struct {
 		filters.BaseSpec `json:",inline"`
 
-		PathPrefix                       *string `json:"pathPrefix,omitempty"`
```

Comment:
Please update the doc in `7.02.filters.md`...

Reply:
Sure, updated